src/css/post-single/_post-single.css

```diff
@@ -67,3 +67,23 @@
 .content h6 {
   margin-bottom: 0;
 }
+
+.promotional {
+  background: #f5d26b;
+  color: #fff;
+  padding: 2rem;
+  margin-bottom: 2rem;
+  /* border: 5px #fff dotted; */
+  text-align: center;
+  border-radius: 2rem;
+}
+.promotional a {
+  color: #fff;
+}
+.promotional strong {
+  text-transform: uppercase;
+  font-size: 2rem;
+}
+.promotional p {
```

Comment:
__[stylelint]__

Expected selector ".promotional p" to come before selector ".content p:first-child" (no-descending-specificity)


<!-- Comment by [Sider](https://sider.review) -->